Tests/FluentAssertions.Shared.Specs/CollectionEquivalencySpecs.cs

```diff
@@ -17,6 +17,128 @@ namespace FluentAssertions.Specs
     [TestClass]
     public class CollectionEquivalencySpecs
     {
+        #region General
```

Comment:
Well, the point is not all the tests in that region tests "Single Object" overload. I splitted them into two regions called "Null (Collection)" and "Null (Single Object" if it's fine.